torch/_torch_docs.py

```diff
@@ -3274,10 +3274,13 @@ def merge_dicts(*dicts):
            r"""
 fmod(input, other, *, out=None) -> Tensor
 
-Computes the element-wise remainder of division.
+Computes the element-wise remainder of division with the remainder having the same
+sign as the divident :attr:`input`.
 
-The dividend and divisor may contain both for integer and floating point
-numbers. The remainder has the same sign as the dividend :attr:`input`.
+.. math::
+    \text{{out}}_i = \text{{input}}_i - tquot * \text{{other}}_i
+
+where :math:`tquot` is the truncated quotient value (rounded towards zero).
```

Comment:
I've added the formula, the concept of `tquot` and `rquot` is inspired from multiple resources. This will help explaining the difference between `fmod` and `remainder`.

Reply:
What would writing out `tquot` look like? `trunc(input / other}`? That might be more helpful than explaining it in English

Reply:
Do these mathematical formulas account for the sign properly? Is a copysign needed?